test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/053_ATF_P_TC_comm_PTU_brings_certificate_RPC_ForceProtectedService_ON.lua

```diff
@@ -0,0 +1,172 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [PTU] [GENIVI] PolicyTableUpdate has NO "certificate" and "ForceProtectedService"=OFF at .ini file
```

Comment:
@dboltovskyi, thank you for notice, done in 5d88bd1b1917f7fb2b273edde450057caf683eee